packages/@okta/vuepress-site/docs/reference/api/domains/index.md

```diff
@@ -67,11 +67,12 @@ curl -v -X POST \
     "validationStatus": "NOT_STARTED",
     "dnsRecords": [
         {
-            "fqdn": "_oktaverification.login.example.com",
+          "recordType": "TXT",
+          "fqdn": "_oktaverification.login.example.com",
             "values": [
                 "79496f234c814638b1cc44f51a782781"
             ],
-            "recordType": "TXT"
+          "expiration": "2021-11-23T02:04:28.000Z"
```

Comment:
since the request example has `MANUAL` we can remove the expiration from the response as currently we are not sending any value for expiration in case of Manual type.